### PR TITLE
Fix autorunner script path

### PR DESCRIPTION
**Pull request by chinzou**

### CODE REVIEW

File: usb-autorunner/.pc/applied-patches [hunk 1]
```diff
@@ -1,2 +1,3 @@
 add-service-scripts
 Update-autorunner-script
+fix-autorunner-script-path
```
Comment (rbuisson) [inline usb-autorunner/.pc/applied-patches:3]: What is this file?

File: usb-autorunner/debian/patches/series [hunk 2]
```diff
@@ -1,2 +1,3 @@
 add-service-scripts
 Update-autorunner-script
+fix-autorunner-script-path
```
Comment (rbuisson) [inline usb-autorunner/debian/patches/series:3]: And this file?

File: usb-autorunner/.pc/applied-patches (hunk 1 above)
Comment (chinzou) [reply usb-autorunner/.pc/applied-patches:3]: it's the patch name apllied to introduce the fix

File: usb-autorunner/debian/patches/series (hunk 2 above)
Comment (chinzou) [reply usb-autorunner/debian/patches/series:3]: This is the list of all available patches

File: usb-autorunner/.pc/applied-patches (hunk 1 above)
Comment (rbuisson) [reply usb-autorunner/.pc/applied-patches:3]: Sure, but why is this here? Who processes this file?

Comment (chinzou) [reply usb-autorunner/.pc/applied-patches:3]: It's used by `dpkg-source` it's the second step in the build process to valid the source of the code in the package

**Pull request merged**